src/r_fps.c

```diff
@@ -207,21 +209,27 @@ void R_UpdateViewInterpolation(void)
 
 		if (oldview_invalid[i]) oldview_invalid[i]--;
 	}
+
+	last_view_update = I_GetTime();
 }
 
 void R_ResetViewInterpolation(UINT8 p)
 {
+	// Wait an extra tic if the interpolation state hasn't
+	// updated yet.
+	int t = last_view_update == I_GetTime() ? 1 : 2;
```

Comment:
This probably should have parentheses around == (`(last_view_update == I_GetTime()) ? 1 : 2;`). Not sure about order of operations here but just to be safe

Reply:
added some there